src/ext/java/org/opentripplanner/ext/transmodelapi/TransmodelIndexGraphQLSchema.java

```diff
@@ -1080,32 +1080,55 @@ public TransmodelIndexGraphQLSchema(Graph graph, RoutingRequest defaultRequest)
                         .name("id")
                         .type(new GraphQLNonNull(Scalars.GraphQLID))
                         .dataFetcher(environment -> relay.toGlobalId(
-                                ptSituationElementType.getName(), ((AlertPatch) environment.getSource()).getId()))
+                                ptSituationElementType.getName(), ((TransitAlert) environment.getSource()).getId()))
```

Comment:
I expect there to be merge conflicts with the current dev-2.x branch for this file. It is ok to accept what ever is in the dev-2.x branch and just comment out the sections that do not compile. We (Entur) can fixit after it is merged.

Reply:
Merge conflicts were fixed, and everything at least compiles succesfully